### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by johnnymo87**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM node:5.7.1
+
+# Install libsass
+RUN apt-get update
+RUN apt-get install -y automake libtool ca-certificates build-essential
+RUN wget https://github.com/sass/libsass/archive/3.3.3.tar.gz && \
```
Comment (tansaku) [inline Dockerfile]: isn't this a css precompiler? https://github.com/sass/libsass why do we need that?

**Comment by tansaku** [comment]
cool!  what I'd really like is a nice way of injecting some environment variables ... I wonder if I can do that through the AWS interface ...

File: Dockerfile (hunk 1 above)
Comment (johnnymo87) [reply Dockerfile]: You're right, we don't.

**Comment by johnnymo87** [comment]
Yeah, I'm not sure how to do environment variables in your setup.

Locally, I sometime use docker-compose to inject environment variables. I recently ditched docker-compose for "raw" docker, and in that case I just pass an `--env-file=` argument to the `docker run` command. In production, we use kubernetes, which does environment variables much like docker-compose.

File: Dockerfile (hunk 1 above)
Comment (tansaku) [reply Dockerfile]: so we can just delete that line, and then feed the docker script into AWS?

https://aws.amazon.com/getting-started/tutorials/deploy-docker-containers/

**Comment by johnnymo87** [comment]
Worth a shot!  Try building it locally first, though.

**Comment by johnnymo87** [comment]
@tansaku I've tinkered locally with the dockerfile, and I believe it works now. I'm not quite sure what `server.js` does. It seems to start and then immediately exit, so maybe I actually don't have it working?

**Comment by tansaku** [comment]
server.js opens a websocket and stays listening on it - i.e. blocking, when I run locally or on AWS ...

**Comment by tansaku** [comment]
@johnnymo87 I just deployed the master branch on dokku (on azure) which uses docker containers under the hood I guess?

**Comment by johnnymo87** [comment]
Sounds good. I'll close this then.

**Pull request closed without merging**